services/scan/src/cli/run_machine.tsx

```diff
@@ -174,20 +174,17 @@ function MachineTextAdventure() {
 
   useInput((input) => {
     switch (input) {
-      // Interpretation mode
-      case 'v':
-        machineService.send({ type: 'SET_INTERPRETATION_MODE', mode: 'valid' });
```

Comment:
If you're removing the interpretation mode setting stuff, maybe just remove the whole thing?

Reply:
I did this and cleaned up a few other things, gonna merge now